sites/docs/content/get-started.md

```diff
@@ -5,11 +5,45 @@ tagline: Guide
 ---
 
 <script>
-	import { Callout } from '$lib/components'
+	import { Steps, Step } from "$lib/components";
 </script>
 
-<Callout type="warning">
+<Steps>
+<Step>Install the package</Step>
 
-The documentation is still a work in progress. Please refer to the [GitHub README](https://github.com/svecosystem/mode-watcher) for more information.
+You can install the project via `npm` or `pnpm`.
 
-</Callout>
+```bash
+pnpm add mode-watcher
+```
+
+<Step>Add the ModeWatcher component</Step>
```

Comment:
I could not get backticks/monspace font to work on the `ModeWatcher` in this step title here. not sure if that's intended